Extension/options.html

```diff
@@ -63,6 +67,11 @@ <h3 style="padding: 0; margin: 0.3em;">Misc:</h3>
     font-family: 'Inter', sans-serif;
     font-size: 0.8em;
   }
+
+  #header {
+    text-align: left;
```

Comment:
Did you upload some old code? This should be center and I've tested that.

Reply:
No, I added a div for the header so the header would be centered and the content wouldn't.

Reply:
No, I'm talking about the text-align: left..
Shouldn't it be center? I tested that and it left aligned the header.